topology/probes/netlink.go

```diff
@@ -492,6 +540,23 @@ func getFamilyKey(family int) string {
 	return ""
 }
 
+func (u *NetNsNetLinkProbe) onRouteChanged(index int64, route *RoutingTable) {
```

Comment:
nit : could be nice to use "rt" for routing table and "route" for a route

Reply:
done